repos/spack_repo/builtin/packages/py_gcovr/package.py

```diff
@@ -26,6 +28,10 @@ class PyGcovr(PythonPackage):
     depends_on("python@2.7:2,3.5:", when="@:4", type=("build", "run"))
     depends_on("py-setuptools", type=("build", "run"))
 
+    depends_on("py-hatchling", when="@8.3:")
+    depends_on("py-hatch-vcs", when="@8.3:")
+    depends_on("py-hatch-fancy-pypi-readme", when="@8.3:")
+
```

Comment:
```suggestion
    depends_on("py-hatchling", type="build", when="@8.3:")
    depends_on("py-hatch-vcs", type="build", when="@8.3:")
    depends_on("py-hatch-fancy-pypi-readme", when="@8.3:", type="build")

```

Let's remove the empty line to join these with existing build dependencies. Also by default dependencies end up as both build and link types and here we only want hatching to be a build dependency.

Reply:
Done